src/thumbnailer.h

```diff
@@ -99,6 +99,11 @@ class Thumbnailer {
     int final_quality = -1;
     float final_psnr;
     bool near_lossless = false;
+    std::pair<int, float>
+        lossy_data[101];  // Array containing pairs of (size, psnr) for
+                          // qualities in range [0..100] when using lossy
+                          // encoding. If WebPEncode() has not been called for
+                          // quality 'x', lossy_data['x'] = (-1,-1.0).
```

Comment:
minor: put the comment above the variable, I think it will be shorter/easier to read this way.